fuel-coroutines/src/test/kotlin/com/github/kittinunf/fuel/coroutines/CoroutinesTest.kt

```diff
@@ -317,4 +321,38 @@ class CoroutinesTest {
             return UUIDResponse(content)
         }
     }
+
+    @Test
+    fun handleException() = runBlocking {
+
+        mock.chain(
+            request = mock.request().withPath("/fail"),
+            response = mock.reflect()
+        )
+
+        val (_, res, result) = Fuel.get(mock.path("fail"))
+            .awaitObjectResponse(object : ResponseDeserializable<Unit> {
+                override fun deserialize(content: String): Unit? {
+                    throw IllegalStateException("some deserialization exception")
+                }
+            })
+
+        Assert.assertThat(res, CoreMatchers.notNullValue())
```

Comment:
[Nits] Can you import `Assert` so it can be used `assertThat` like others

Reply:
done